unison-cli/src/Unison/Codebase/Editor/HandleInput/Update.hs

```diff
@@ -242,6 +243,15 @@ getSlurpResultForUpdate requestedNames slurpCheckNames = do
     -- terms in it fails.
     let slurp0 = slurp unisonFile0
 
+    -- Grab some interim info out of the original slurp.
```

Comment:
Maybe we can expand a little bit about why this interim info, on Weds

Reply:
Agreed, I'd love to know what the "iterim" info is used for, and if possible incorporate that description in the name or docs.

Reply:
The "interim" info is used all over the place, it's simply the name I picked for the the just-typechecked file that seeds this whole algorithm of discovering more terms from the codebase that form components with the updated ones (including component-mates and all terms along all paths from new version to old version). It doesn't have to be let-bound; we could have just as easily reached into the `hashTermsId` field of the latest typechecked unison file of the slurp result. So, I guess what I mean is - I'm not sure that this particular let-binding would be the place to clear up any confusion resulting from its choice of name. Is there something specific about the algorithm I could document better? (I assume "yes" - just call out as appropriate)